x-pack/plugins/triggers_actions_ui/public/application/lib/get_defaults_for_action_params.ts

```diff
@@ -29,5 +29,11 @@ export const getDefaultsForActionParams = (
         pagerDutyDefaults.eventAction = EventActionOptions.RESOLVE;
       }
       return pagerDutyDefaults;
+    case '.xmatters':
```

Comment:
@pmuellr If I remember correctly there is an issue about defaults: https://github.com/elastic/kibana/issues/117169

Reply:
Not clear to me if that issue is resolved now that we have variables available for recovered.  Or if this is still an issue after that, is there something else needed here?